sys-libs/libxcrypt/libxcrypt-4.4.19.ebuild

```diff
@@ -102,19 +147,25 @@ src_test() {
 src_install() {
 	multibuild_foreach_variant multilib-minimal_src_install
 
+	if ! use system; then
 	(
 		shopt -s failglob || die "failglob failed"
 
 		# Make sure our man pages do not collide with glibc or man-pages.
-		for manpage in "${ED}"/usr/share/man/man3/crypt{,_r}.?*; do
+		for manpage in "${ED}$(get_xcmandir)"/man3/crypt{,_r}.?*; do
 			mv -n "${manpage}" "$(dirname "${manpage}")/xcrypt_$(basename "${manpage}")" \
 				|| die "mv failed"
 		done
 	) || die "failglob error"
+	fi
 
 	# remove useless stuff from installation
-	find "${D}"/usr/share/doc/${PF} -type l -delete || die
 	find "${D}" -name '*.la' -delete || die
+	if is_cross; then
```

Comment:
?

Reply:
My problem here was that I could not find any way to remove the doc files from the standard `/usr/share/doc/libxcrypt` path which caused file conflicts between cross-* builds and plain sys-libs/libxcrypt, so I ended up removing those files from cross builds as you see above.

I tried a lot of methods including passing `--docdir` in the cross-* configure step, but that didn't work even though `./configure` documentation says it should work. For example `--mandir` works as documented and there are no conflicts there between cross an non-cross builds.

BTW, as a side-note, the `sys-apps/man-pages` package also provides the following files which conflict with `sys-apps/libxcrypt`, my solution there was to install the libxcrypt documentation in a subdirectory because --mandir works. Will post this in the next iteration unless I can come up with something better.

```
/usr/share/man/man3/crypt.3.bz2
/usr/share/man/man3/crypt_r.3
```

Any suggestion is welcome. Maybe we could remove the crypt documentation from the man-pages package if it is disabled in glibc?

I'll also try to see if I can somehow get `--docdir` to work, to avoid deleting the cross doc.